### PR TITLE
Add Travis CI

### PR DESCRIPTION
**Pull request by mattmahn**
The app will be tested against Node 6 (normal release) and 4 (LTS). (I put v4 in there just so we can see pass/fail; we can remove v4 if it causes us bother.) Travis _should_ post to our `#code` Slack channel when:
- Fail &rarr; Success
- all Failures

**P.S.** Who ever merges this should probably clean up my commit history by squashing.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+  - '6'
+  - '4'
+notifications:
+  email: false
+  slack:
+    secure: NGn87q+6FfLVKwuC0j0IbqT5uJgZUTEiSZwSMzZnVpnI0f27sT4cRQDEJpfKge9GJoMyPUNR8rfYJVX5zSLEV+jXV6sCxnQO3PDNSPJyvBjQuuiNF/ZNItEc+TEg3QCiv2s9PjcVQfc2r7F6+tVG5Ce+vfTpsgRZplfAG3PAYtSePHUTbWhDvARq5eTMW++P1oqaxhL0KxGuQMKBqjU2S+LAt1ky7UdhZqu6+6LY1oaaTDjl8Yvn/GdRklD39aWW6vx2Z2kvuFW2Stq9vtQ0GFZdt8ttTbn2l8nwAuOluIFvseooTtClDfcVDutQD42TD5/pRGjSHSaNmGbMCEcMKxQ55e4gY5KUkWdEMXIq0u9gr5xQa63pq/tQoSni/9YOs4jtFLYYU0ZWr6zY6cpqka7q8uxE22P61abLTIlEbSGatNsTgKoX5Upnj2U/L8BKbcWG4YHwXdlG6F7qyMIxTArYd9aja3nTLgXzwhT2f0WgwwB+vDv5LoQHNm2H2VXrS4rwQ5lhD5DGzFvQFl8WS+p7EHLibHN2s+ZiU8k+LvVN88THDlun+G5jVAoJAkg83OVguU284BZC+6kHbxMtjMm3T9lgAAXp+mcWaL0Jopm+JTlRvnT5hKyJVklKtWB0Gn4ox73qsSAc7OrX2P7ZrZ6ztITWNA6LdPquQVYZBYA=
```
Comment (trevinhofmann) [inline .travis.yml:8]: What's this token? Looks like something that should be private.

File: .editorconfig [hunk 2]
```diff
@@ -8,5 +8,5 @@ charset = utf-8
 indent_style = space
 indent_size = 4
 
-[package.json]
+[{package.json,.travis.yml}]
```
Comment (trevinhofmann) [inline .editorconfig:11]: :+1:

File: README.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # consus-flux
 
+[![Build Status](https://travis-ci.org/TheFourFifths/consus-flux.svg?branch=master)](https://travis-ci.org/TheFourFifths/consus-flux)
```
Comment (trevinhofmann) [inline README.md:3]: :+1:

File: .travis.yml (hunk 1 above)
Comment (trevinhofmann) [reply .travis.yml:8]: Ok, this seems to be encrypted. Just want to make sure.

Comment (mattmahn) [reply .travis.yml:8]: Yup, its our encrypted Slack token.

**Review by trevinhofmann — APPROVED** [review]

**Comment by trevinhofmann** [comment]
ACK

**Pull request merged**